docs/DeveloperGuide.md

```diff
@@ -6,9 +6,6 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
-
-
 ## Product scope
 ### Target user profile
```

Comment:
Perhaps more user stories can better explain the features and application of your product.